src/wranglerAction.ts

```diff
@@ -424,6 +424,36 @@ async function wranglerCommands(
 					setOutput("pages-deployment-alias-url", pagesArtifactFields.alias);
 					setOutput("pages-deployment-id", pagesArtifactFields.deployment_id);
 					setOutput("pages-environment", pagesArtifactFields.environment);
+					// create github deployment, if GITHUB_TOKEN is provided
+					if (
```

Comment:
This logic for outputs is already not very friendly. I don't feel great about tangling up the github deployments with it. Could we add in some separation?